pages/api/camara/v1/partidos/[partido].js

```diff
@@ -0,0 +1,36 @@
+import microCors from 'micro-cors';
+import getPartido from '../../../../../services/camara/partido';
```

Comment:
Se não me engano também temos path alias no projeto, então daria pra fazer algo mais ou menos assim:
```suggestion
import getPartido from '@/services/camara/partido';
```